src/libs/antares/study/area/links.cpp

```diff
@@ -711,8 +711,8 @@ bool AreaLinksLoadFromFolder(Study& study, AreaList* l, Area* area, const AnyStr
             {
                 // Copper plate mode
                 auto infinity = +std::numeric_limits<double>::infinity();
-                link.directCapacities.fillColumn(0, +infinity);
-                link.indirectCapacities.fillColumn(0, +infinity);
+                link.directCapacities.fill(+infinity);
```

Comment:
There 2 symbols **"+"** here : one in the definition of **infinity**, and one when using **infinity** as argument of the **fill** function. Not sure the second one is required.
But otherwise, ok !